main.tex

```diff
@@ -130,7 +147,7 @@ \subsection*{Co-op}
 \clearpage
 
 
-\section{Setup}
+\addsection{SETUP}{\images/skulls.png}
 This section will guide you through the process of setting up a scenario from the mission book.
```

Comment:
```suggestion
\addsection{SETUP}{\images/skulls.png}
\bigbreak
This section will guide you through the process of setting up a scenario from the mission book.
```
I think those headers need a little more spacing.

Without `\bigbreak`:
![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/ced8cb54-7a1c-4b71-8e7d-ded9efa8bb43)

With:
![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/b17f213a-1218-462a-887a-bc59b9524530)

Reply:
Actually, you can make it a part of your function instead of typing it every time.